test/parameter-names-convention.test.js

```diff
@@ -57,12 +69,14 @@ test('az-parameter-names-convention should find errors', () => {
     },
   };
   return linter.run(oasDoc).then((results) => {
-    expect(results.length).toBe(5);
+    expect(results.length).toBe(7);
     expect(results[0].path.join('.')).toBe('paths./test1/{test-id}.parameters.0.name');
     expect(results[1].path.join('.')).toBe('paths./test1/{test-id}.parameters.1.name');
     expect(results[2].path.join('.')).toBe('paths./test1/{test-id}.parameters.2.name');
-    expect(results[3].path.join('.')).toBe('paths./test1/{test-id}.get.parameters.0.name');
-    expect(results[4].path.join('.')).toBe('paths./test1/{test-id}.get.parameters.1.name');
+    expect(results[3].path.join('.')).toBe('paths./test1/{test-id}.parameters.3.name');
+    expect(results[4].path.join('.')).toBe('paths./test1/{test-id}.parameters.4.name');
```

Comment:
Since this rule can produce multiple messages, we should make sure we are getting the one we want.
```suggestion
    expect(results[3].path.join('.')).toBe('paths./test1/{test-id}.parameters.3.name');
    expect(results[3].message).toContain("should not begin with '$' or '@'");
    expect(results[4].path.join('.')).toBe('paths./test1/{test-id}.parameters.4.name');
    expect(results[4].message).toContain("should not begin with '$' or '@'");
```